components/Nav.jsx

```diff
@@ -1,81 +1,91 @@
-                            LMS Platform
+                    <a className="hidden lg:block items-center p-2 mr-4 ">
+                        <span className="text-2xl mr-2 text-black font-bold uppercase tracking-wide">
+                            LMS
+                        </span>
+                        <span className="text-xl text-black tracking-wide">
+                            Platform
                         </span>
                     </a>
                 </Link>
                 <button
-                    className=" inline-flex p-3 hover:bg-green-600 rounded lg:hidden text-white ml-auto hover:text-white outline-none"
+                    className=" inline-flex p-3 hover:bg-purple-500 rounded lg:hidden text-white  hover:text-white outline-none"
                     onClick={handleClick}
                 >
-                    <svg
-                        className="w-6 h-6"
-                        fill="none"
-                        stroke="currentColor"
-                        viewBox="0 0 24 24"
-                        xmlns="http://www.w3.org/2000/svg"
-                    >
-                        <path
-                            strokeLinecap="round"
-                            strokeLinejoin="round"
-                            strokeWidth={2}
-                            d="M4 6h16M4 12h16M4 18h16"
-                        />
-                    </svg>
+                    <span className="text-2xl mr-2 text-black font-bold uppercase tracking-wide">
+                        LMS
+                    </span>
+                    <span className="text-xl text-black tracking-wide">
+                        Platform
+                    </span>
                 </button>
                 {/*Note that in this div we will use a ternary operator to decide whether or not to display the content of the div  */}
+                <div className="lg:hidden ml-auto">
+                    <Icon
+                        icon={bxNotification}
+                        style={{ color: '#0f172a', fontSize: '30px' }}
+                    />
+                    <Avatar src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png" />
+                </div>
                 <div
                     className={`${
                         active ? '' : 'hidden'
                     }   w-full lg:inline-flex lg:flex-grow lg:w-auto`}
                 >
                     <div className="lg:inline-flex lg:flex-row lg:ml-auto lg:w-auto w-full lg:items-center items-start  flex flex-col lg:h-auto">
                         <Link href="/">
-                            <a className="lg:inline-flex lg:w-auto w-full px-3 py-2 rounded text-white font-bold items-center justify-center hover:bg-green-600 hover:text-white ">
+                            <a className="lg:inline-flex lg:w-auto w-full mx-8 px-3 py-1 rounded text-black  items-center justify-center hover:bg-purple-500 hover:bg-opacity-50 hover:text-white ">
```

Comment:
Could we maybe change the purple to match the side nav color? some type of blue

Reply:
And also dev layout was merged into dev, ik you merged ryans branch but just double check that's its ok

Reply:
i sent the blues in msteams but ill send it here too.  try `bg-blue-600` and `hover:bg-blue-500`

Reply:
@ryan-hong  I am just wondering where should bg-blue-600 be applied?